t/lib/WTSI/NPG/HTS/Illumina/LogPublisherTest.pm

```diff
@@ -7,6 +7,9 @@ use Carp;
 use English qw[-no_match_vars];
 use Log::Log4perl;
 use Test::More;
+use File::Temp qw(tempdir);
+use File::Copy::Recursive qw(dircopy);
+use File::Spec;
```

Comment:
If any of these three packages are not in the 'requires' or 'build_requires' sections of Build.PL, please add them to the 'build_requires' section.